apps/tailwind-components/assets/css/main.css

```diff
@@ -128,6 +131,10 @@
     --text-color-pagination-input: var(--text-color-title);
     --text-color-pagination-hover: var(--text-color-title);
     --text-color-footer-link: var(--color-yellow-500);
+    --text-color-listbox: var(--color-black);
```

Comment:
I am still struggling to understand what should be defined in this main.css and what on component level.

Given our most recent discussion that we won't reproduce a bootstrap css I get impression that most of the time it should be on component level?